content/id/docs/kegiatan-eastern-townhall-indonesia.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Kegiatan Eastern Townhall Indonesia"
+linkTitle: "Kegiatan Eastern Townhall Indonesia"
+hide_feedback: true
+weight: 20
+# description:
+---
+> *Sebagai informasi, kegiatan ini khusus di Eastern Townhall (ETH) Indonesia saja dan dimulai sejak tanggal 10 Jan 2022*
```

Comment:
made this into a note block.

Reply:
Ah, you've pushed the revised version

Reply:
Ok, it's merged, just saw your Discord message.